apiserver/pkg/util/cluster_test.go

```diff
@@ -351,7 +351,7 @@ func TestBuildVolumes(t *testing.T) {
 						AccessModes: []corev1.PersistentVolumeAccessMode{
 							corev1.ReadWriteOnce,
 						},
-						Resources: corev1.ResourceRequirements{
+						Resources: corev1.VolumeResourceRequirements{
```

Comment:
This may be a breaking change https://github.com/kubernetes/kubernetes/pull/118653. We may revert the change if users complain it.